hr_applicant/views/hr_recruitment_views.xml

```diff
@@ -0,0 +1,325 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_hr_applicant_medical_details_form">
+        <field name="name">view.hr.applicant.medical.details.form</field>
+        <field name="model">hr.applicant.medical.details</field>
+        <field name="arch" type="xml">
+            <form string="Medical Screening" version="9.0">
```

Comment:
@ilakkia-serpentcs remove all `version="9.0"`

Reply:
removed all  version="9.0" and added report record based on v15